backend/src/auth/auth.module.ts

```diff
@@ -1,10 +1,31 @@
 import { Module } from '@nestjs/common';
+import { APP_GUARD } from '@nestjs/core';
+import { AuthController } from './auth.controller';
 import { AuthGuard } from './auth.guard';
+import { AuthService } from './auth.service';
 import { RolesGuard } from './roles.guard';
+import { TokenGuard } from './token.guard';
 import { LoggerModule } from '../logger/logger.module';
+import { PersonnelModule } from '../personnel/personnel.module';
 
 @Module({
-  imports: [LoggerModule],
-  providers: [AuthGuard, RolesGuard],
+  imports: [LoggerModule, PersonnelModule],
+  providers: [
+    AuthService,
```

Comment:
Moved these from main into the module in order to avoid explicitly passing in the reflector.